gradle.properties

```diff
@@ -1,4 +1,4 @@
-version = 0.1.1-SNAPSHOT
+version = 0.1.1
```

Comment:
Please don't bump version in the feature PR. Let's have releases in separate PRs for clarity of git history.
Also, let's keep master head always on `-SNAPSHOT`

Reply:
Done